docs/dev/policy_documents/merging-and-promoting-conventions.md

```diff
@@ -0,0 +1,40 @@
+# Merging and Promoting Conventions
+
+## Summary
+[summary]: #summary
+
+This policy defines a 'fast-track' merge and promote policy for a subset of core-plans.  
+
+## Motivation
+[motivation]: #motivation
+
+If someone who contributes heavily to core plans, has a pull request for a minor change open for 7days +, with all tests passing, it would be good if the same person can also merge and promote if there is a lack of peer reviews. One contributor (Graham Wilson, predominant) has hundreds of updates in his backlog, but has been holding back because of delays in the review process.
+
+## Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+What plan is eligable for the fast-track criteria:
+
+* Only identified 'veteran' core-plan contributor's have permission to use this fast-track policy.  This list will be agreed and specified in the MAINTAINERS.md
+* This fasttrack will only apply to non-baseplan PRs and PRs that do not have a label 'DO-NOT-MERGE' or are otherwise marked for caution.
+* If no feedback has been added to the PR after 7+ days, then the veteran contributor can merge and promote the PR.
```

Comment:
Suggested Addition:

* The PR consists of minor changes. For example, version bumps, style improvements, bug fixes that don't change the scope of the plan.

Minor style suggestion: 

I think it might be better to rephrase these eligibility requirements in the form of positive assertions that can be checked:

- PR was opened by 'veteran' core-plan contributor. Veteran core-plan contributors are specified in MAINTAINERS.md;
- PR is for a non-base plan and does not have the 'DO-NOT-MERGE' label;
- All automated testing has passed; and,
- PR has not received feedback for 7+ days.

Reply:
Thanks @stevendanna. Will add these.